.github/workflows/test_and_deploy.yml

```diff
@@ -50,14 +50,14 @@ jobs:
       run: "python -m pytest --import-mode=append -svx nle/tests"
 
   test_sdist:
-    name: Test sdist on MacOS w/ Py3.8
+    name: Test sdist on MacOS w/ Py3.11
```

Comment:
I think there is some value in testing the oldest version supported (3.8 takes the "most work" in some sense). But at the same time there's value in testing the newest version too.

Reply:
Ok, good idea. I've added another test in that workflow for 3.8. The 3.11 one will be the one distributed.